### DISCUSSION TITLE
Welcome to BalancingWii Discussions!

### DISCUSSION THREAD

**Discussion by mahowik** [mahowik/BalancingWii#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satpalsony** [comment]
Now I am busy with some other work, I will definitely take time to try and
thank you very much

**Comment by harshad52** [comment]
I'm using a reversal shaft stepper motors, nema17, but when controlled through app it's not spinning it's stuck at one place not balancing at all
Also checked the motor drivers they are working fine , any solution to fix
Waiting for your reply.........

**Comment by Annoniteinc** [comment]
Is gps function capable to be use on the balancing robot?

**Comment by pzavoli71** [comment]
Can't understand how "calculateSubperiods" function operates. Can, someone, explain the details of the purpose of:
```
  subperiodPadding = ((1000 % absSpeed)*8)/absSpeed;
  if (subperiodPadding > 0) {
    subPeriod[motor][1]++;
  }  
  if (subperiodPadding > 1) {
    subPeriod[motor][5]++;
  }  
  if (subperiodPadding > 2) {
    subPeriod[motor][3]++;
  }  
  if (subperiodPadding > 3) {
    subPeriod[motor][7]++;
  }  
  if (subperiodPadding > 4) {
    subPeriod[motor][0]++;
  }  
  if (subperiodPadding > 5) {
    subPeriod[motor][4]++;
  }  
  if (subperiodPadding > 6) {
    subPeriod[motor][2]++;
  }  
```
